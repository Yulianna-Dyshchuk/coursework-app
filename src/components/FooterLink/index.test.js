import React from "react";
import { shallow } from "enzyme";
import { FooterLink } from "./index";

describe("FooterLink component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<FooterLink />);

    expect(component).toMatchSnapshot();
  });

  it("should render correctly with props", () => {
    const component = shallow(<FooterLink type="test" label="test" to="test" target="test" />);

    expect(component).toMatchSnapshot();
  });
});
