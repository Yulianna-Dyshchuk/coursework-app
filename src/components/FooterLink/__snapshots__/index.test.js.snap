// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLink component should render correctly with props 1`] = `
<a
  href="test"
  target="test"
  type="test"
>
  <WithStyles(ForwardRef(Typography))
    variant="body2"
  >
    test
  </WithStyles(ForwardRef(Typography))>
</a>
`;

exports[`FooterLink component should render correctly without props 1`] = `
<Link
  to=""
>
  <WithStyles(ForwardRef(Typography))
    variant="body2"
  />
</Link>
`;
